mycore-base/src/test/java/org/mycore/datamodel/objectinfo/MCRObjectInfoEntityQueryResolverTest.java

```diff
@@ -88,8 +89,9 @@ private static void initClassifications() throws SAXParseException, MalformedURL
         DAO = new MCRCategoryDAOImpl();
         CLS = MCRCategLinkServiceFactory.getInstance();
 
-        Document xml = MCRXMLParserFactory.getParser()
-            .parseXML(new MCRURLContent(new URL("https://mycore.de/classifications/mir_licenses.xml")));
+        URL classResourceUrl = MCRCategoryDAOImplTest.class
+            .getResource("/" + MCRCategoryDAOImplTest.class.getSimpleName() + "/mir_licenses.xml");
```

Comment:
Should use `MCRObjectInfoEntityQueryResolverTest.class` in both places.

Reply:
I know that this would require another copy of the (possibly truncated) `mir_licenses.xml`, but as I see it, the latest precedent is, that every test class should have its own resource folder with its own resources. If that means duplicating some stuff, that that is the natural side effect of this approach and should be followed through consistently.